spring-integration-amqp/src/main/java/org/springframework/integration/amqp/dsl/AmqpOutboundChannelAdapterSpec.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.amqp.dsl;
+
+import org.springframework.amqp.core.AmqpTemplate;
+import org.springframework.messaging.Message;
+
+/**
+ * Spec for an outbound AMQP channel adapter
+ *
+ * @author Gary Russell
+ * @since 5.3
+ *
+ */
+public class AmqpOutboundChannelAdapterSpec extends AmqpOutboundEndpointSpec<AmqpOutboundChannelAdapterSpec> {
+
+	AmqpOutboundChannelAdapterSpec(AmqpTemplate amqpTemplate) {
```

Comment:
Gary, can you make all the ctors and `final` props in these AMQP specs as `protected` for possible extension?
So, I won't include them into my extensions PR #3167 to avoid conflicts between our two fixes.

Thanks